<?xml version="1.0" encoding="UTF-8"?>
<testng-results>
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-07-02T19:13:00Z" name="Surefire suite" finished-at="2019-07-02T19:13:05Z" duration-ms="4126">
    <groups>
    </groups>
    <test started-at="2019-07-02T19:13:00Z" name="Surefire test" finished-at="2019-07-02T19:13:05Z" duration-ms="4126">
      <class name="raul.reactive.streams.publisher.SimplePublisherTest">
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:00Z" name="setUp" finished-at="2019-07-02T19:13:00Z" duration-ms="5" status="PASS">
        </test-method>
        <test-method signature="optional_spec104_mustSignalOnErrorWhenFails()" started-at="2019-07-02T19:13:00Z" name="optional_spec104_mustSignalOnErrorWhenFails" finished-at="2019-07-02T19:13:00Z" duration-ms="1" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.
	at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1193)
	at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1182)
	at org.reactivestreams.tck.PublisherVerification.optional_spec104_mustSignalOnErrorWhenFails(PublisherVerification.java:361)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:00Z" name="setUp" finished-at="2019-07-02T19:13:00Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="optional_spec105_emptyStreamMustTerminateBySignallingOnComplete()" started-at="2019-07-02T19:13:00Z" name="optional_spec105_emptyStreamMustTerminateBySignallingOnComplete" finished-at="2019-07-02T19:13:01Z" duration-ms="115" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="optional_spec111_maySupportMultiSubscribe()" started-at="2019-07-02T19:13:01Z" name="optional_spec111_maySupportMultiSubscribe" finished-at="2019-07-02T19:13:01Z" duration-ms="102" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront()" started-at="2019-07-02T19:13:01Z" name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront" finished-at="2019-07-02T19:13:01Z" duration-ms="105" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Did not receive expected element or completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Did not receive expected element or completion within 100 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1226)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1167)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront(PublisherVerification.java:654)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected()" started-at="2019-07-02T19:13:01Z" name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected" finished-at="2019-07-02T19:13:01Z" duration-ms="108" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Did not receive expected element or completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Did not receive expected element or completion within 100 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1226)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1167)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected(PublisherVerification.java:686)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne()" started-at="2019-07-02T19:13:01Z" name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne" finished-at="2019-07-02T19:13:01Z" duration-ms="108" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@1aa7ecca did not produce the requested 2 elements on 2nd subscriber within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@1aa7ecca did not produce the requested 2 elements on 2nd subscriber within 100 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1226)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1167)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne(PublisherVerification.java:604)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="optional_spec111_registeredSubscribersMustReceiveOnNextOrOnCompleteSignals()" started-at="2019-07-02T19:13:01Z" name="optional_spec111_registeredSubscribersMustReceiveOnNextOrOnCompleteSignals" finished-at="2019-07-02T19:13:01Z" duration-ms="107" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Did not receive expected stream completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Did not receive expected stream completion within 100 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1226)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1167)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_registeredSubscribersMustReceiveOnNextOrOnCompleteSignals(PublisherVerification.java:577)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="optional_spec309_requestNegativeNumberMaySignalIllegalArgumentExceptionWithSpecificMessage()" started-at="2019-07-02T19:13:01Z" name="optional_spec309_requestNegativeNumberMaySignalIllegalArgumentExceptionWithSpecificMessage" finished-at="2019-07-02T19:13:01Z" duration-ms="104" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipped because tested publisher does NOT implement this OPTIONAL requirement.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1226)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1165)
	at org.reactivestreams.tck.PublisherVerification.optional_spec309_requestNegativeNumberMaySignalIllegalArgumentExceptionWithSpecificMessage(PublisherVerification.java:920)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="required_createPublisher1MustProduceAStreamOfExactly1Element()" started-at="2019-07-02T19:13:01Z" name="required_createPublisher1MustProduceAStreamOfExactly1Element" finished-at="2019-07-02T19:13:01Z" duration-ms="107" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Did not receive expected stream completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Did not receive expected stream completion within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:169)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.expectCompletion(TestEnvironment.java:1000)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.expectCompletion(TestEnvironment.java:511)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.requestEndOfStream(TestEnvironment.java:425)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.requestEndOfStream(TestEnvironment.java:412)
	at org.reactivestreams.tck.PublisherVerification$1.run(PublisherVerification.java:167)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_createPublisher1MustProduceAStreamOfExactly1Element(PublisherVerification.java:162)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="required_createPublisher3MustProduceAStreamOfExactly3Elements()" started-at="2019-07-02T19:13:01Z" name="required_createPublisher3MustProduceAStreamOfExactly3Elements" finished-at="2019-07-02T19:13:01Z" duration-ms="106" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@1a482e36 produced only 1 element expected:<true> but was:<false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@1a482e36 produced only 1 element expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at org.reactivestreams.tck.PublisherVerification$2.run(PublisherVerification.java:184)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_createPublisher3MustProduceAStreamOfExactly3Elements(PublisherVerification.java:179)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:01Z" name="setUp" finished-at="2019-07-02T19:13:01Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="required_spec101_subscriptionRequestMustResultInTheCorrectNumberOfProducedElements()" started-at="2019-07-02T19:13:01Z" name="required_spec101_subscriptionRequestMustResultInTheCorrectNumberOfProducedElements" finished-at="2019-07-02T19:13:02Z" duration-ms="313" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@4f83df68 produced less than 3 elements after two respective `request` calls within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@4f83df68 produced less than 3 elements after two respective `request` calls within 100 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:232)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:959)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.nextN(TestEnvironment.java:988)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:484)
	at org.reactivestreams.tck.PublisherVerification$3.run(PublisherVerification.java:224)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec101_subscriptionRequestMustResultInTheCorrectNumberOfProducedElements(PublisherVerification.java:211)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.AssertionError: Publisher org.reactivestreams.FlowAdapters$ReactivePublisherFromFlow@4f83df68 produced less than 3 elements after two respective `request` calls within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:229)
	... 36 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec102_maySignalLessThanRequestedAndTerminateSubscription()" started-at="2019-07-02T19:13:02Z" name="required_spec102_maySignalLessThanRequestedAndTerminateSubscription" finished-at="2019-07-02T19:13:02Z" duration-ms="106" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Did not receive expected element or completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Did not receive expected element or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:232)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:959)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.nextN(TestEnvironment.java:988)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:484)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:472)
	at org.reactivestreams.tck.PublisherVerification$4.run(PublisherVerification.java:244)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec102_maySignalLessThanRequestedAndTerminateSubscription(PublisherVerification.java:239)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.AssertionError: Did not receive expected element or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:229)
	... 37 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec105_mustSignalOnCompleteWhenFiniteStreamTerminates()" started-at="2019-07-02T19:13:02Z" name="required_spec105_mustSignalOnCompleteWhenFiniteStreamTerminates" finished-at="2019-07-02T19:13:02Z" duration-ms="105" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Did not receive expected element within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Did not receive expected element within 100 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:232)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:959)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElement(TestEnvironment.java:456)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.requestNextElement(TestEnvironment.java:391)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.requestNextElement(TestEnvironment.java:382)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.requestNextElement(TestEnvironment.java:378)
	at org.reactivestreams.tck.PublisherVerification$7.run(PublisherVerification.java:402)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec105_mustSignalOnCompleteWhenFiniteStreamTerminates(PublisherVerification.java:397)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.AssertionError: Did not receive expected element within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:229)
	... 38 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="required_spec107_mustNotEmitFurtherSignalsOnceOnCompleteHasBeenSignalled()" started-at="2019-07-02T19:13:02Z" name="required_spec107_mustNotEmitFurtherSignalsOnceOnCompleteHasBeenSignalled" finished-at="2019-07-02T19:13:02Z" duration-ms="207" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Did not receive expected stream completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Did not receive expected stream completion within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:169)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.expectCompletion(TestEnvironment.java:1000)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.expectCompletion(TestEnvironment.java:511)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.expectCompletion(TestEnvironment.java:499)
	at org.reactivestreams.tck.PublisherVerification$9.run(PublisherVerification.java:436)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec107_mustNotEmitFurtherSignalsOnceOnCompleteHasBeenSignalled(PublisherVerification.java:430)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe()" started-at="2019-07-02T19:13:02Z" name="required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe" finished-at="2019-07-02T19:13:02Z" duration-ms="3" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.
	at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1193)
	at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1182)
	at org.reactivestreams.tck.PublisherVerification.required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe(PublisherVerification.java:520)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="required_spec109_mustIssueOnSubscribeForNonNullSubscriber()" started-at="2019-07-02T19:13:02Z" name="required_spec109_mustIssueOnSubscribeForNonNullSubscriber" finished-at="2019-07-02T19:13:02Z" duration-ms="2" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec109_subscribeThrowNPEOnNullSubscriber()" started-at="2019-07-02T19:13:02Z" name="required_spec109_subscribeThrowNPEOnNullSubscriber" finished-at="2019-07-02T19:13:02Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec302_mustAllowSynchronousRequestCallsFromOnNextAndOnSubscribe()" started-at="2019-07-02T19:13:02Z" name="required_spec302_mustAllowSynchronousRequestCallsFromOnNextAndOnSubscribe" finished-at="2019-07-02T19:13:02Z" duration-ms="103" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec303_mustNotAllowUnboundedRecursion()" started-at="2019-07-02T19:13:02Z" name="required_spec303_mustNotAllowUnboundedRecursion" finished-at="2019-07-02T19:13:02Z" duration-ms="103" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to validate call stack depth safety, awaited at-most 2 signals (`maxOnNextSignalsInRecursionTest()`) or completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to validate call stack depth safety, awaited at-most 2 signals (`maxOnNextSignalsInRecursionTest()`) or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:169)
	at org.reactivestreams.tck.TestEnvironment$Latch.expectClose(TestEnvironment.java:852)
	at org.reactivestreams.tck.PublisherVerification$19.run(PublisherVerification.java:819)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec303_mustNotAllowUnboundedRecursion(PublisherVerification.java:752)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec306_afterSubscriptionIsCancelledRequestMustBeNops()" started-at="2019-07-02T19:13:02Z" name="required_spec306_afterSubscriptionIsCancelledRequestMustBeNops" finished-at="2019-07-02T19:13:02Z" duration-ms="104" status="PASS">
        </test-method>
        <test-method signature="required_spec307_afterSubscriptionIsCancelledAdditionalCancelationsMustBeNops()" started-at="2019-07-02T19:13:02Z" name="required_spec307_afterSubscriptionIsCancelledAdditionalCancelationsMustBeNops" finished-at="2019-07-02T19:13:03Z" duration-ms="107" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:02Z" name="setUp" finished-at="2019-07-02T19:13:02Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec309_requestNegativeNumberMustSignalIllegalArgumentException()" started-at="2019-07-02T19:13:03Z" name="required_spec309_requestNegativeNumberMustSignalIllegalArgumentException" finished-at="2019-07-02T19:13:03Z" duration-ms="103" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="required_spec309_requestZeroMustSignalIllegalArgumentException()" started-at="2019-07-02T19:13:03Z" name="required_spec309_requestZeroMustSignalIllegalArgumentException" finished-at="2019-07-02T19:13:03Z" duration-ms="105" status="PASS">
        </test-method>
        <test-method signature="required_spec312_cancelMustMakeThePublisherToEventuallyStopSignaling()" started-at="2019-07-02T19:13:03Z" name="required_spec312_cancelMustMakeThePublisherToEventuallyStopSignaling" finished-at="2019-07-02T19:13:03Z" duration-ms="107" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec313_cancelMustMakeThePublisherEventuallyDropAllReferencesToTheSubscriber()" started-at="2019-07-02T19:13:03Z" name="required_spec313_cancelMustMakeThePublisherEventuallyDropAllReferencesToTheSubscriber" finished-at="2019-07-02T19:13:03Z" duration-ms="328" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue()" started-at="2019-07-02T19:13:03Z" name="required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue" finished-at="2019-07-02T19:13:03Z" duration-ms="104" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="required_spec317_mustSupportACumulativePendingElementCountUpToLongMaxValue()" started-at="2019-07-02T19:13:03Z" name="required_spec317_mustSupportACumulativePendingElementCountUpToLongMaxValue" finished-at="2019-07-02T19:13:03Z" duration-ms="102" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Did not receive expected element or completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Did not receive expected element or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:232)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:959)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.nextN(TestEnvironment.java:988)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:484)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:472)
	at org.reactivestreams.tck.PublisherVerification$29.run(PublisherVerification.java:1061)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec317_mustSupportACumulativePendingElementCountUpToLongMaxValue(PublisherVerification.java:1053)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.AssertionError: Did not receive expected element or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:229)
	... 37 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="required_spec317_mustSupportAPendingElementCountUpToLongMaxValue()" started-at="2019-07-02T19:13:03Z" name="required_spec317_mustSupportAPendingElementCountUpToLongMaxValue" finished-at="2019-07-02T19:13:04Z" duration-ms="107" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Did not receive expected element or completion within 100 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Did not receive expected element or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:232)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:959)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.nextN(TestEnvironment.java:988)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:484)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElements(TestEnvironment.java:472)
	at org.reactivestreams.tck.PublisherVerification$28.run(PublisherVerification.java:1041)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification.required_spec317_mustSupportAPendingElementCountUpToLongMaxValue(PublisherVerification.java:1035)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.AssertionError: Did not receive expected element or completion within 100 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:229)
	... 37 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:03Z" name="setUp" finished-at="2019-07-02T19:13:03Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:04Z" name="setUp" finished-at="2019-07-02T19:13:04Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="required_validate_boundedDepthOfOnNextAndRequestRecursion()" started-at="2019-07-02T19:13:04Z" name="required_validate_boundedDepthOfOnNextAndRequestRecursion" finished-at="2019-07-02T19:13:04Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="required_validate_maxElementsFromPublisher()" started-at="2019-07-02T19:13:04Z" name="required_validate_maxElementsFromPublisher" finished-at="2019-07-02T19:13:04Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:04Z" name="setUp" finished-at="2019-07-02T19:13:04Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:04Z" name="setUp" finished-at="2019-07-02T19:13:04Z" duration-ms="1" status="PASS">
        </test-method>
        <test-method signature="stochastic_spec103_mustSignalOnMethodsSequentially()" started-at="2019-07-02T19:13:04Z" name="stochastic_spec103_mustSignalOnMethodsSequentially" finished-at="2019-07-02T19:13:05Z" duration-ms="1008" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed in iteration 0 of 100. Expected completion signal after signalling 10 elements (signalled 1), yet did not receive it within 1000 ms]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed in iteration 0 of 100. Expected completion signal after signalling 10 elements (signalled 1), yet did not receive it within 1000 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:169)
	at org.reactivestreams.tck.TestEnvironment$Latch.expectClose(TestEnvironment.java:852)
	at org.reactivestreams.tck.PublisherVerification$5$1.run(PublisherVerification.java:347)
	at org.reactivestreams.tck.PublisherVerification.activePublisherTest(PublisherVerification.java:1138)
	at org.reactivestreams.tck.PublisherVerification$5.apply(PublisherVerification.java:258)
	at org.reactivestreams.tck.PublisherVerification$5.apply(PublisherVerification.java:255)
	at org.reactivestreams.tck.PublisherVerification.stochasticTest(PublisherVerification.java:1207)
	at org.reactivestreams.tck.PublisherVerification.stochastic_spec103_mustSignalOnMethodsSequentially(PublisherVerification.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled()" started-at="2019-07-02T19:13:05Z" name="untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled(PublisherVerification.java:425)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled()" started-at="2019-07-02T19:13:05Z" name="untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled(PublisherVerification.java:446)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals()" started-at="2019-07-02T19:13:05Z" name="untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals(PublisherVerification.java:451)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice()" started-at="2019-07-02T19:13:05Z" name="untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice" finished-at="2019-07-02T19:13:05Z" duration-ms="1" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice(PublisherVerification.java:550)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="untested_spec109_subscribeShouldNotThrowNonFatalThrowable()" started-at="2019-07-02T19:13:05Z" name="untested_spec109_subscribeShouldNotThrowNonFatalThrowable" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec109_subscribeShouldNotThrowNonFatalThrowable(PublisherVerification.java:456)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="untested_spec304_requestShouldNotPerformHeavyComputations()" started-at="2019-07-02T19:13:05Z" name="untested_spec304_requestShouldNotPerformHeavyComputations" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec304_requestShouldNotPerformHeavyComputations(PublisherVerification.java:832)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="setUp()" started-at="2019-07-02T19:13:05Z" name="setUp" finished-at="2019-07-02T19:13:05Z" duration-ms="0" status="PASS">
        </test-method>
        <test-method signature="untested_spec305_cancelMustNotSynchronouslyPerformHeavyComputation()" started-at="2019-07-02T19:13:05Z" name="untested_spec305_cancelMustNotSynchronouslyPerformHeavyComputation" finished-at="2019-07-02T19:13:05Z" duration-ms="1" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Not verified by this TCK.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1212)
	at org.reactivestreams.tck.PublisherVerification.untested_spec305_cancelMustNotSynchronouslyPerformHeavyComputation(PublisherVerification.java:837)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
